api/app/routers/collection.py

```
@@ -0,0 +1,760 @@
+
+        return {
+            "message": "Collection item soft deleted",
+            "id": collection_id,
+            "is_active": collection_item.is_active,
+        }
+
+
+@router.post("/bulk", response_model=BulkCollectionResponse)
+async def bulk_collection_operations(
+    request: BulkCollectionRequest,
+    current_user: CurrentUser = Depends(get_current_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)